site/docs/hosting/vps.md

```diff
@@ -201,6 +201,49 @@ systemd is a powerful service manager which is pre-installed on many Linux distr
    - `WantedBy=multi-user.target` --- defines the system state in which the service should be launched.
      `multi-user.target` --- is a typical value for servers.
 
+   :::: tip Using the `.env` file
```

Comment:
I think we're using title case for tips, so it should be
```suggestion
   :::: tip Using the `.env` File
```

Reply:
Yes, yes. I always forget about it 😅